src/orange/orangeinp/ConvexRegion.cc

```diff
@@ -356,9 +356,9 @@ GenTrap GenTrap::from_trap(
     CELER_VALIDATE(theta >= zero_quantity() && theta < Turn{0.25},
                    << "invalid angle " << theta.value()
                    << " [turns]: must be in the range [0, 0.25)");
-    CELER_VALIDATE(phi >= zero_quantity() && phi < Turn{1.},
+    CELER_VALIDATE(phi > Turn{-1} && phi < Turn{1},
```

Comment:
This is probably enough:
```suggestion
    CELER_VALIDATE(phi > Turn{-0.5} && phi <= Turn{0.5},
```
and it matches the image range of atan2 function.

Reply:
True... maybe we should update the inf wedge to have the same bounds (I think I fmod it between 0 and 1). But since we just take trig functions of the value, anything is valid honestly... i could imagine a user calling with 3/4 instead of -1/4